platform/darwin/src/MGLRendererConfiguration.h

```diff
@@ -35,6 +35,19 @@ NS_ASSUME_NONNULL_BEGIN
  which will be available at run time, e.g. "PingFang". */
 @property (nonatomic, readonly) mbgl::optional<std::string> localFontFamilyName;
 
+/**
+ A Boolean value indicating whether symbol layers may enable cross-source symbol
+ collision detection.
+ 
+ Set `MGLCrossSourceCollisions` in your containing app's Info.plist
+ 
+ Setting this property to `NO`, symbol layers will only run collision detection
+ against other symbol layers that are part of the same source.
+ 
+ The default value of this property is `YES`.
```

Comment:
It’s a bit awkward that leaving the Info.plist option unset is equivalent to `YES`. Normally leaving an option unset would be equivalent to `NO`. Could we rename the option to make it more naturally have `NO` as a default value? Perhaps `MGLPerSourceCollisions` or `MGLV3CollisionBehavior`.